test/conf.js

```diff
@@ -0,0 +1,61 @@
+exports.config = {
+
+	// setting to launch tests directly without selenium server
+	directConnect : true,
+	// address of running selenium server
+	seleniumAddress : 'http://localhost:4444/wd/hub',
+
+	// setting time outs
+	getPageTimeOut : 200000,
+	allScriptsTimeout : 500000,
+
+	// setting framework
+	framework : 'custom',
+	frameworkPath : require.resolve('protractor-cucumber-framework'),
+
+	// setting protractor to ignore uncaught exceptions to take care by protractor-cucumber-framework
+	ignoreUncaughtExceptions : true,
+
+	// configuration parameters
+	params: {
+		testEnv: 'test'
+    },
```

Comment:
Please check all javascript files meet the style guid by running `standard on them`

E.g. `./admin/node_modules/.bin/standard test/**/*.js`